### PR TITLE
Add webtest.2.0.0 and webtest-js.2.0.0

### PR DESCRIPTION
**Pull request by johnelse**

### CODE REVIEW

**Comment by johnelse** [comment]
Looks like the travis failure is `js_of_ocaml` failing to build with OCaml 4.11

**Comment by mseri** [comment]
Do you know if the js version compiles with 4.11 using maybe a dev version of js_of_ocaml or if we need an upper bound?

File: packages/webtest-js/webtest-js.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" ] {with-test}
+]
+
+depends: [
+  "dune" {>= "1.1"}
+  "js_of_ocaml" {>= "3.4.0"}
+  "js_of_ocaml-ppx" {>= "3.4.0"}
+  "ocaml" {>= "4.02.1"}
+  "webtest"
```
Comment (mseri) [inline packages/webtest-js/webtest-js.2.0.0/opam:18]: We tend to use stricter versioning when packages are released together. Would you mind if we add the following constraint? Could you also port it upstream?

```suggestion
  "webtest" {=version}
```

Comment (johnelse) [reply packages/webtest-js/webtest-js.2.0.0/opam]: Sure! I've added this change.

**Comment by mseri** [comment]
Thanks!

**Comment by johnelse** [comment]
Thank you!

**Pull request merged**